### ISSUE TITLE
Fix slow response time warnings

### ISSUE THREAD

**Issue by seanh** [hypothesis/via#128]
We're getting repeated slow response time warnings from Via 3 in the #eng-monitoring channel and given that we're expecting an increase in traffic soon, any scalability or performance issues with Via 3 need to be fixed.

If there's actually a problem here it needs to be fixed. If it's a false alarm then the alarm needs to be fixed.

See Slack thread: https://hypothes-is.slack.com/archives/C0LUWQQJJ/p1586212841006700

New relic alert: https://rpm.newrelic.com/accounts/1385283/applications/358378127?tw%5Bstart%5D=1586210225&tw%5Bend%5D=1586212025

----

There is a multi-stage plan in place to try and address this:

 * [x] Change Via 3 to assume Google Drive documents are PDFs 
    - https://github.com/hypothesis/via3/pull/131
 * [x] Change LMS to assume Google Drive documents are PDFs and skip Via 3's routing 
    - https://github.com/hypothesis/lms/pull/1685
 * ~Add NGINX to New relic monitoring to check if we are having problems there too (blocked on a purchase right now)~ This has a separate ticket: https://github.com/hypothesis/via3/issues/21 and is currently blocked by Newrelic not wanting money
 * [x] Change LMS to redirect ~all~ Canvas documents it knows should be a PDF straight to Via 3 display, rather than repeatedly triggering the content-type detection and routing
    - https://github.com/hypothesis/lms/pull/1686

Other ideas which are nice to haves:

 * Potentially attempt to handle Google Drive and other error cases a bit better by:
    * Being able to redirect to a page saying "We don't support that document type" for docs and the like
   * Checking for Google Drive document mime-types via API to allow us to push you to the correct page - https://github.com/hypothesis/via3/pull/130

**Comment by jon-betts** [comment]
We've got the important bits here to do with lowering load and increasing speed. There were some extra parts about monitoring, but this duplicated another ticket. There were also some ideas about a nicer user experience for errors.

TBH our error presentation in Via3 (or Via for that matter) is not good right now full stop. When we switch to HTML, we will need to know the mime-type for documents, and we have an API key setup now, so that can probably happen as and when.
